### DISCUSSION TITLE
DreamBerd vHas vBeen vRenamed cTo nDreamBerd

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#484] [Announcements]
vWelcome cTo adjAnother adjWeekly nDreamBerd nRelease nNotes.

## adjNew nName

aFirst, pWe'vRe qReally adjPleased cTo vAnnounce nDreamberd's adjNew nName: nDreamBerd.

pWe vPromise pWe vWon't vChange pIt advAgain.

## adjSemantic nNaming

pWe'vRe qAlso adjExcited cTo vIntroduce aA qBrand adjNew nFeature. pWe vCall pIt adjSemantic vNaming, cOr adjHungarian nNotation.

advNow pIt's adjEasier cTo vRead nDreamberd cThan advEver nBefore.

```java
const const sName = "Lu"!
const const iAge = 29!
const const bHappy = true!
```

pWe qEven vSupport g_nGlobals.

```java
const const g_fScore = 4.5!
```

pWe vLook advForward cTo vSeeing vYou cFor adjNext nWeek's nRelease nNotes.

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/DreamBerd/releases/tag/vv1.0'>DreamBerd vHas vBeen vRenamed cTo nDreamBerd</a>.</em>

**Comment by skifli** [comment]
detThis advSounds adjIncredible pun!

**Reply by theprogrammingbird** [reply]
yeah

**Comment by mybearworld** [comment]
pYou vSeem cTo vHave vMade artA nMistake - cIf nDreamberd vHas vBeen vRenamed, vWouldn't pIt vBe nNdreamberd?

**Comment by D-u-c-k-s-e-l** [comment]
detThis nWebsite vShould vConsider vUsing adjA adjDifferent nFont. adjThe adjUppercase nI cnjAnd aLowercase nL vLook prpToo advSimilar. pI vThought pYou vSaid nPlt.

**Reply by mybearworld** [reply]
`pI vHave artA nSolution.`

**Reply by D-u-c-k-s-e-l** [reply]
> `pI vHave artA nSolution.`

pWhat vIs pIt?

(intjNoo! pYou vMade pMe vSay pIt prpToo!)

## nEDIT:

```
pI vSee pYour nIdea advNow!
```

**Reply by Furlanetto-Dev** [reply]
pI vCan vRead pThis nShit

**Comment by mybearworld** [comment]
iHow vDo pYou vSay "that's" pWith adjSemantic nNaming? "dThat's", "dThatv'S" pOr nSomething cElse advEntirely?

**Reply by MaddyGuthridge** [reply]
pMy adjPersonal nPreference vIs "dThat'vS" cAs pIt vKeeps dThe adjSemantic nMarker adjClosest preTo pIts nWord

**Reply by TodePond** [reply]
aThe adjBest nThing proAbout adjSemantic nNaming vIs cThat nEveryone vDoes pIt adjInconsistently.

**Comment by MaddyGuthridge** [comment]
iHave pYou vConsidered vUpdating dThe nKeywords infmTo vReflect dThis nChange? advCurrently, dThe nLack preOf adjSemantic nNaming vMakes dThose nParts preOf dThe nSyntax adjVery adjDifficult infmTo vUnderstand. pI'vM vThinking dThat pIts nUsage vCould nBe nSomething preLike pThis:

* `vConst`
* `nVar`
* `nFunction`
* `cIf`
* advEtc...

**Comment by GoldenStack** [comment]
No

**Reply by D-u-c-k-s-e-l** [reply]
```
vDon't vSay n"No", vSay n"intjNo".
pThis vIs adjMore nFun. (pI vThink....?)
```

**Reply by loglot** [reply]
> vSay n"intjNo".

intjNo.

**Comment by Furlanetto-Dev** [comment]
aOMG pThat adjHurts pMy pBrain
